janus_core/calculations/md.py

```diff
@@ -713,6 +761,9 @@ def run(self) -> None:
         if self.post_process_kwargs:
             self._post_process()
 
+        if self.correlation_kwargs:
+            self._write_correlations()
```

Comment:
It's not really a problem, but why is this handled differently to parsing/attaching (checking for self.correlation_kwargs here, rather than a check within the function)?